packages/chart/src/RadialBar.ts

```diff
@@ -0,0 +1,167 @@
+            .delay((d, i) => i * this.rowDelay())
+            .duration(this.tweenDuration())
+            .attrTween('d', (d, i) => {
+                let interpolate = d3interpolate(0, d[1]);
+                return t => this.arc(interpolate(t), i);
+            });
+        this.arcs
+            .selectAll('.arc-text')
+            .data(this.data())
+            .enter().append('text')
+            .attr('class', 'arc-text');
+        this.svg.selectAll('.arc-text')
+            .attr("x", -4)
+            .attr("y", (d, i) => -(this.getInnerRadius(i) + this.getOuterRadius(i)) / 2)
+            .attr("alignment-baseline", "middle")
+            .attr("text-anchor", "end")
+            .text(d => d[0])
+    }
+    getInnerRadius(index) {
+        return this.arcMinRadius() + (this.data().length - (index + 1)) * (this.arcWidth + this.arcPadding());
+    }
+    getOuterRadius(index) {
+        return this.getInnerRadius(index) + this.arcWidth;
+    }
+    rad2deg(angle) {
+        return angle * 180 / Math.PI;
+    }
+    //  INDChart
+    _palette;
+    fillColor: (row, column, value) => string;
+    strokeColor: (row, column, value) => string;
+    textColor: (row, column, value) => string;
+    click: (row, column, selected) => void;
+    dblclick: (row, column, selected) => void;
+    //  ITooltip
+    tooltip;
+    tooltipHTML: (_) => string;
+    tooltipFormat: (_) => string;
+    _selection;
+}
+RadialBar.prototype._class += " chart_RadialBar";
+RadialBar.prototype.implements(INDChart.prototype);
+RadialBar.prototype.implements(ITooltip.prototype);
+RadialBar.prototype.mixin(Utility.SimpleSelectionMixin);
+export interface RadialBar {
+    paletteID(): string;
+    paletteID(_: string): this;
+    radiusPadding(): number;
+    radiusPadding(_: number): this;
+    maxRange(): number;
+    maxRange(_: number): this;
+    maxDomainMultiplier(): number;
+    maxDomainMultiplier(_: number): this;
+    rowDelay(): number;
+    rowDelay(_: number): this;
+    tweenDuration(): number;
+    tweenDuration(_: number): this;
+    tickCount(): number;
+    tickCount(_: number): this;
+    arcPadding(): number;
+    arcPadding(_: number): this;
+    arcMinRadius(): number;
+    arcMinRadius(_: number): this;
+}
+RadialBar.prototype.publish("paletteID", "default", "set", "paletteID", RadialBar.prototype._palette.switch());
+RadialBar.prototype.publish("radiusPadding", 40, "number", "Padding between chart edge and container edge (px)", RadialBar.prototype._palette.switch());
+RadialBar.prototype.publish("rowDelay", 100, "number", "Delay between the start of each row animation (ms)");
+RadialBar.prototype.publish("tweenDuration", 1000, "number", "Duration of a row's animation (ms)");
+RadialBar.prototype.publish("maxRange", 2 * Math.PI, "number", "Maximum angular length of a bar (radians)");
+RadialBar.prototype.publish("maxDomainMultiplier", 1.62, "number", "Multiplied by the highest value to determine the maximum arc length (>= 1.0)");
```

Comment:
Not sure what this is doing?  Looks like its multiplying the max "value" not domain by 1.62?  I suspect its to prevent the bar from going a full 360 degrees?

Reply:
Right, it provides a buffer for the upper limit of the domain. `.maxDomainMultiplier(1)` would display the largest value as a 360 arc.